src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,74 @@
+package duke.parser;
+
+import duke.command.*;
```

Comment:
I think it would be better if the import statements are explicitly mentioned, rather than using *, as it becomes much clearer for others to see and understand.